lessonPlans/views.py

```diff
@@ -32,6 +32,13 @@ def topic_detail(request, topic_id):
     try:
         topic = Topic.objects.get(pk = topic_id)
         description = topic.description
+        current_user = request.user
+        plans = []
+
+        for plan in LessonPlan.objects.all():
+            if current_user in plan.assigned_users.all() and plan.topics.filter(name = topic.name):
```

Comment:
Does `plan.topics.filter(name = topic.name)` return boolean?

Reply:
Empty array in python is type converted to a falsy value.

Reply:
It is not a good practice. Use `topic.name in plan.topics.all()` instead.

Reply:
This will always return false, I've tried it.
![image](https://user-images.githubusercontent.com/54092258/118443096-1d5aa300-b709-11eb-84b6-8ba6532d8f2f.png)
The output of this is :
![image](https://user-images.githubusercontent.com/54092258/118443286-55fa7c80-b709-11eb-8daf-e52827c17024.png)
if you insist on using ``` plan.topics.all() ``` we'll have iterate over this (see the "this is successful" loop) as this returns a Querselector array which has all the fields on a lesson plan and not just the name, I think filter is the better option as it removes the need for that extra for loop.
Correct me if I'm wrong though I am just speaking on the basis of debugging it.

Reply:
My bad, it should be `topic in plan.topics.all()`

Reply:
Done 👍🏻